sys-boot/gnu_efi/gnu_efi-3.0.4.recipe

```diff
@@ -0,0 +1,31 @@
+SUMMARY="GNU EFI Library - Kernel support"
+DESCRIPTION="
+Develop EFI applications for ARM-64, ARM-32, x86_64, IA-64 (IPF), and IA-32 (x86)
+platforms using the GNU toolchain and the EFI development environment.
+"
+HOMEPAGE="https://sourceforge.net/projects/gnu-efi/"
+COPYRIGHT="
+	1999-2007 Hewlett-Packard Co.
+	2006-2010 Intel Co.
+	"
+LICENSE="BSD (3-clause)"
+SOURCE_URI="http://downloads.sourceforge.net/project/gnu-efi/gnu-efi-$portVersion.tar.bz2"
+CHECKSUM_SHA256="51a00428c3ccb96db24089ed8394843c4f83cf8f42c6a4dfddb4b7c23f2bf8af"
+SOURCE_DIR="gnu-efi-$portVersion"
+REVISION="1"
+ARCHITECTURES="any"
+DISABLE_SOURCE_PACKAGE="yes"
+
+PROVIDES="
+	gnu_efi_kernel = $portVersion compat >= 0
```

Comment:
Don't forget to change PROVIDES to
```
PROVIDES="
	gnu_efi = $portVersion compat >= 0
	"
```
as @korli wrote earlier.

BTW, to handle @korlis's other request, you should:
```
cd haikuports/sys-boot
mkdir -p gnu-efi
git mv gnu_efi/gnu_efi-3.0.4.recipe gnu-efi/gnu_efi-3.0.4.recipe
git commit --amend --no-edit gnu_efi/gnu_efi-3.0.4.recipe gnu-efi/gnu_efi-3.0.4.recipe
```